### ISSUE TITLE
PHP 7.4.13 is not available on windows.php.net

### ISSUE THREAD

**Issue by alvin0319** [pmmp/php-build-scripts#115]
# Full compile log
```

[PocketMine] PHP Windows compiler 
[PocketMine] Setting up environment... 
[PocketMine] Building release binaries with debug symbols 
[PocketMine] Using path C:\pocketmine-php-sdk for build sources 
[PocketMine] Found Visual Studio 2017 
**********************************************************************
** Visual Studio 2017 Developer Command Prompt v15.9.30
** Copyright (c) 2017 Microsoft Corporation
**********************************************************************
[vcvarsall.bat] Environment initialized for: 'x64'
[PocketMine] Getting SDK... 
Note: switching to 'cb51b8f716d826e465685e5a589523ede6487f76'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

[PocketMine] Downloading PHP source version 7.4.13... 
[PocketMine] [ERROR] Failed to download PHP source - check compile.log for details 
```

![image](https://user-images.githubusercontent.com/32565818/104080866-01805580-526e-11eb-90bd-d2d1f4dc1f4d.png)

**Comment by fuyutsuki** [comment]
Change `windows-compile-vs.bat` to the following
```diff
- set PHP_VER=%PHP_MAJOR_VER%.13
+ set PHP_VER=%PHP_MAJOR_VER%.14
```

**Comment by dktapps** [comment]
Fixed by 17dd1a3ff86e07b81bfd98e9c01f70b6951ce199.
